keps/693-multikueue/README.md

```diff
@@ -189,9 +189,22 @@ const (
     SecretLocationType LocationType = "Secret"
 )
 
-type MultiKueueClusterSpec {
-    // Information how to connect to the cluster.
-    KubeConfig KubeConfig `json:"kubeConfig"`
+// +kubebuilder:validation:ExactlyOneOf=kubeConfig;clusterProfile
+
+type MultiKueueClusterSpec struct {
+    // Information about how to connect to the cluster.
+    // Exactly one of KubeConfig or ClusterProfile must be specified.
```

Comment:
nit, move the comment above the declaration of `type MultiKueueClusterSpec struct {`